core/eth/tx.go

```diff
@@ -236,13 +245,18 @@ func (t *Transactor) RegisterOperatorWithChurn(ctx context.Context, pubkeyG1 *co
 		t.Logger.Error("Failed to generate transact opts", "err", err)
 		return err
 	}
-	tx, err := t.Bindings.BLSRegCoordWithIndices.RegisterOperatorWithCoordinator(
+
+	// TODO(mooselumph): Fill out these
+	kickParams := []regcoordinator.IRegistryCoordinatorOperatorKickParam{}
```

Comment:
is this to be done in following PR? can we create a ticket to track it

Reply:
Adding a comment to the ticket for the churner updates: https://linear.app/eigenlabs/issue/EGDA-194/churner-updates-for-m2-mainnet-contracts#comment-66163e52